internal/store/deployment.go

```diff
@@ -73,6 +73,21 @@ func deploymentMetricFamilies(allowLabelsList []string) []generator.FamilyGenera
 				}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_deployment_status_ready_replicas",
```

Comment:
Could we name the metric `kube_deployment_status_replicas_ready` to make it consistent with the other deployment metrics, as well as with https://github.com/kubernetes/kube-state-metrics/blob/eba9142028841d5181ce09836fb1e0c2cf54f90c/internal/store/statefulset.go#L91

Reply:
@fpetkovski 
Fixed the metric name
Thank you for pointing this out